### ISSUE TITLE
Block list is ignored

### ISSUE THREAD

**Issue by gingerbeardman** [apexskier/DefaultBrowser#24]
Could be related to #22 

## Problem

- I have certain apps blocked but this list is not honoured

<img width="442" alt="Screen shot 2022-12-06 at 14 00 35" src="https://user-images.githubusercontent.com/49612/205932342-27a1257b-2772-468c-8f3f-11455e496a4b.png">

## Expected
- they should not show in the previous browsers list of Default Browser

## Actual
- they have begun appearing in the previous browsers list of Default Browser

<img width="210" alt="Screen shot 2022-12-06 at 14 00 41" src="https://user-images.githubusercontent.com/49612/205932353-2fe7a8aa-f587-436c-8b74-4935e2854d01.png">

- Default Browser 1.3.1
- macOS 12.6.1 (21G217)

**Comment by gingerbeardman** [comment]
I believe the design of the app is such that:

1. all browser apps are listed in the menu
2. blocked apps are not sent the URLs

I've been using it such a long time I should really know.

But this means that now that #22 is solved blocklist should be honoured.

TL;DR closing.
